database/migrations/20221102-create-role.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    await queryInterface.createTable('Role', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      name: {
+        type: Sequelize.STRING
+      },
+      description: {
+        type: Sequelize.STRING
+      },
+    });
```

Comment:
### Pending

Please add the createdAt and the updatedAT

Reply:
pudiste ver si te llego el cambio¡

Reply:
No se ha actualizado el comió